packages/wrangler/src/__tests__/pages/pages-deployment-tail.test.ts

```diff
@@ -20,15 +22,41 @@ import type {
 import type { RequestInit } from "undici";
 import type WebSocket from "ws";
 
+vi.mock("ws", async (importOriginal) => {
```

Comment:
This has been moved out of `vitest.setup.ts` since we only need to mock `ws` in two files